src/integration/java/io/pinecone/helpers/TestIndexResourcesManager.java

```diff
@@ -26,8 +25,8 @@ public class TestIndexResourcesManager {
             ? "us-east4-gcp"
             : System.getenv("PINECONE_ENVIRONMENT");
     private static final String metric = System.getenv("METRIC") == null
-            ? IndexMetric.COSINE.toString()
-            : System.getenv("METRIC");
+            ? IndexMetric.DOTPRODUCT.toString()
```

Comment:
Remind me why we want this to be dotproduct again?

Reply:
Indexes must be dotproduct in order to allow querying sparse vectors: https://docs.pinecone.io/guides/data/understanding-hybrid-search#limitations